server/routes/__tests__/check-your-details.spec.js

```diff
@@ -0,0 +1,129 @@
+const { getServer } = require('../../../.jest/setup')
+const {
+  submitGetRequest,
+  submitPostRequest
+} = require('../../__test-helpers__/server')
+const mockPolygons = require('../../services/__data__/mockPolygons.json')
+
+jest.mock('../../services/agol/getFloodZones')
+jest.mock('../../services/riskAdmin/isRiskAdminArea')
+jest.mock('../../services/agol/getContacts')
```

Comment:
const mockPolygons = require('../../services/__data__/mockPolygons.json')
jest.mock('../../services/agol/getFloodZones')
jest.mock('../../services/riskAdmin/isRiskAdminArea')
 
could be replaced by 
const { mockPolygons } = require('../../services/__tests__/__mocks__/floodZonesByPolygonMock')